README.md

```diff
@@ -116,3 +116,29 @@ If you need additional proxies, you can run them on separate consoles:
 Finally, you can disable the proxy if you don't need it at all:
 
     roslaunch rvizweb rvizweb.launch interactive_markers:=false
+
+## Viewing depth clouds
+
+To view depth clouds, you will need [web_video_server](https://wiki.ros.org/web_video_server) and [depthcloud_encoder](https://wiki.ros.org/depthcloud_encoder) running.
+
+Here's a basic example using Turtlebot:
+
+1. Install `turtlebot_gazebo`:
+
+        sudo apt install -y ros-kinetic-turtlebot-gazebo
+
+1. Run `turtlebot_world` in Gazebo:
+
+        roslaunch turtlebot_gazebo turtlebot_world.launch
+
+1. Launch RVizWeb enabling depth clouds:
+
+        roslaunch rvizweb rvizweb.launch depth_cloud:=true
```

Comment:
Did you need to change the global frame or the depth cloud frame for the turtlebot example to look ok? For me, the only frames available seem to be `odom` and `base_footprint`, and after trying a few combinations, the cloud still looked rotated:

![depthcloudrotated](https://user-images.githubusercontent.com/5751272/50026096-8bc93300-ff9c-11e8-8c3d-2239d118b28d.png)